src/main/java/bookstore/model/Category.java

```diff
@@ -25,6 +25,7 @@ public class Category {
     @Column(nullable = false, unique = true)
     private String name;
 
+    @Column
```

Comment:
Why do you need it?

Reply:
As I said in the previous pull request - for uniformity and appearance